File: 151004/Stahnov/rest/Beans/src/main/java/com/distributed_computing/bean/Creator.java
package com.distributed_computing.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Creator {
    int id;
    String login;
    String password;
    String firstname;
    String lastname;
}

File: 151003/Alekseeva/TaskKafka/Module.Publisher/Models/Creator.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;
namespace Publisher.Models;

[Table("tbl_creators")]
[Index(nameof(Login), IsUnique = true)]
public class Creator
{
    public long Id { get; set; }

    [MaxLength(64)] public string Login { get; set; } = string.Empty;

    [MaxLength(128)] public string Password { get; set; } = string.Empty;

    [MaxLength(64)] public string FirstName { get; set; } = string.Empty;

    [MaxLength(64)] public string LastName { get; set; } = string.Empty;
    public List<Tweet> Tweets { get; } = new();
}